### PR TITLE
ICU-22723 CLDR 46 beta1 to main

### PR DESCRIPTION
**Pull request by DraganBesevic**
##### Checklist

- [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-22723 
- [ ] Required: The PR title must be prefixed with a JIRA Issue number. 
- [ ] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [ ] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [ ] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

Beta 1 integration for CLDR 46 that changes since alpha release.

Minor updates to unit tests to account for changes made for ISO 8601

https://unicode-org.atlassian.net/browse/CLDR-17892

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by richgillam — APPROVED** [review]
RSLGTM

**Comment by markusicu** [comment]
> Azure Pipelines failed to run 1 pipeline(s).

I am surprised that Azure is responding at all, after @rp9-next deleted all of the workflows there. Is there some GitHub-Azure connection still up?

File: icu4c/source/data/brkitr/root.txt [hunk 1]
```diff
@@ -25,5 +25,5 @@ root{
     lstm{
         Thai{"Thai_graphclust_model4_heavy.res"}
         Mymr{"Burmese_graphclust_model5_heavy.res"}
-    }
+    }
```
Comment (markusicu) [inline icu4c/source/data/brkitr/root.txt:28]: why the trailing spaces?

**Review by markusicu — APPROVED** [review]

**Comment by DraganBesevic** [comment]
> > Azure Pipelines failed to run 1 pipeline(s).
> 
> I am surprised that Azure is responding at all, after @rp9-next deleted all of the workflows there. Is there some GitHub-Azure connection still up?

I asked about this in an email thread and was pointed to https://unicode-org.github.io/icu/userguide/dev/ci.html#exhaustive-tests

However, I don't see the "Exhaustive Tests" workflow in my fork. So, I am not sure how to run them. Wouldn't be better to have it available here, in the PR, instead of going outside of the flow (and maybe mistime it) ?

**Comment by markusicu** [comment]
> However, I don't see the "Exhaustive Tests" workflow in my fork. So, I am not sure how to run them. Wouldn't be better to have it available here, in the PR, instead of going outside of the flow (and maybe mistime it) ?

We discussed that, and it was expensive and cumbersome, so we decided that it's good enough to document it as a manual workflow. @echeran @mihnita @rp9-next for details.

**Review by pedberg-icu — APPROVED** [review]

**Comment by rp9-next** [comment]
> I am surprised that Azure is responding at all, after @rp9-next deleted all of the workflows there. Is there some GitHub-Azure connection still up?

I haven't yet deleted the pipelines from Azure dashboards yet. Also, I still need to delete the connection between Azure and this GitHub repo. I thought of doing it in maybe a month once we were content that we no longer needed the pipelines.





> > However, I don't see the "Exhaustive Tests" workflow in my fork. So, I am not sure how to run them. Wouldn't be better to have it available here, in the PR, instead of going outside of the flow (and maybe mistime it) ?
> 
> We discussed that, and it was expensive and cumbersome, so we decided that it's good enough to document it as a manual workflow. @echeran @mihnita @rp9-next for details.

Yes, we tried to mimic the azure behaviour of triggering the workflow on a comment, but in the GHA workflows, we would end up with a lot of stale workflow requests (on each comment) that would get cancelled. So, we decided to keep things simple, by triggering the workflow manually.




> However, I don't see the "Exhaustive Tests" workflow in my fork.

Apologies for any confusion, as stated in the discussion, it is caused because GitHub expects the workflow file to be present in the default branch to show up in the fork. Syncing your fork's main branch should make the workflow appear. Also, created #3175 for improving the docs

**Review by mihnita — APPROVED** [review]
LGTM for the icu4j subfolder.

Thank you,
Mihai

**Comment by DraganBesevic** [comment]
> > However, I don't see the "Exhaustive Tests" workflow in my fork. So, I am not sure how to run them. Wouldn't be better to have it available here, in the PR, instead of going outside of the flow (and maybe mistime it) ?
> 
> We discussed that, and it was expensive and cumbersome, so we decided that it's good enough to document it as a manual workflow. @echeran @mihnita @rp9-next for details.

Just as a reference, Exhaustive Tests passed for this branch in my fork

https://github.com/DraganBesevic/icu/actions/runs/10925509839

**Pull request merged**